lib/gds_api/email_alert_api.rb

```diff
@@ -247,6 +247,24 @@ def send_subscription_verification_email(address:, frequency:, topic_id:)
     )
   end
 
+  # Unsubscribe all users for a subscription list
+  # optionally send a notification email explaining the reason
+  #
+  # @param [string]               subscriber_list_id    SubscriptionList slug, for example a content_id for single page notifications
```

Comment:
> for example a content_id for single page notifications

No, it's the ID of the subscriber list.

Reply:
ah of course, sorry got that mixed up with the other one.